tests/rbd/rbd_ecpool_test_image.py

```diff
@@ -0,0 +1,85 @@
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def test_rbd_ecpool(**kw):
+    """Verify image creation and deletion on ecpools.
+
+    This module verifies image operations on ecpools
+
+    Args:
+        kw: test data
+            config:
+                ec-pool-k-m: <k,m>
+                rep_pool_name: <name>
+                ec_pool_name: <name>
+
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    """
+    rbd = Rbd(**kw)
+    pool = kw["config"].get("pool", rbd.random_string())
+    image = kw["config"].get("image", rbd.random_string())
+    size = kw["config"].get("image_size", "10G")
+
+    try:
+        if not rbd.create_pool(poolname=pool):
+            # create pool does not catch exceptions, it returns true/false.
```

Comment:
we can use initial_rbd_config method once PR https://github.com/red-hat-storage/cephci/pull/1991 is merged.

Reply:
Done.